sdk/src/main/java/com/hedera/hashgraph/sdk/Node.java

```diff
@@ -19,6 +27,23 @@
         this.attempts = 0;
     }
 
+    @Nullable
+    public NetworkName getNetworkName() {
+        return networkName;
+    }
+
+    public Node setNetworkName(NetworkName networkName) {
+        this.networkName = networkName;
+        return this;
+    }
+
+    @Override
+    ChannelCredentials getChannelCredentials() {
+        return TlsChannelCredentials.newBuilder()
+            .trustManager(new HederaTrustManager(networkName, accountId))
```

Comment:
We should have an integration test that verifies TLS works.

Reply:
When this PR was merged the was no way to "enable" TLS for a previous network which would make creating a TLS integration test difficult; you'd need to set the network yourself using `Client.forNetwork()` where the IPs were `:50212`. Since the IPs change per previewnet/testnet hard coding them into an integration test seems like a bad idea. Nonetheless, once we add support for switching to TLS through some configuration we should definitely add a test for it.